### ISSUE TITLE
Unable to trade with new villagers

### ISSUE THREAD

**Issue by Koji-KAWAHARA** [samedifferent/morevillagers#21]
### What's the issue you encountered?
A villager with new profession respond like an unemployed one when right clicked, showing no trading GUI.
They don't trade anything.

### How can the issue be reproduced?
Simply install the mod, place down a new workbench and make a new employed villager, right click them.
This issue is also true for villagers in naturally generated villages.

### Mod Version
Forge 36.2.2 (MC 1.16.5)
More villagers-FORGE-1.16.5-1.5.4.jar
(No other mod installed)

### Images, Videos, or Logs
Non

**Comment by samedifferent** [comment]
Does this happen with all of the villagers, or just certain ones? It is also helpful to see your log

**Comment by Koji-KAWAHARA** [comment]
Thanks for the response.
Happened for all type of the villagers.
Here's the log files of me create a new creative world, place all workbenches and villagers, check that they don't trade, and quit.

[debug.log](https://github.com/samedifferent/morevillagers/files/6913118/debug.log)
[latest.log](https://github.com/samedifferent/morevillagers/files/6913119/latest.log)

**Comment by Koji-KAWAHARA** [comment]
Also, I tested with older versions of forge (forge 36.1.16 and 36.1.32), and the problem stayed.
Not sure if this helps tho.

**Comment by samedifferent** [comment]
I've thoroughly tested the latest version of More Villagers on both singleplayer and on a server, in survival and creative, and they're trading just fine. Have you tried using a different launcher or making a new instance of Minecraft?

**Comment by Koji-KAWAHARA** [comment]
Thanks to your comment, I got it working now!
Seems like this problem was a AdoptOpenJDK version issue. I use mojang launcher, and used to run on adoptopenjdk 11.0.9.11-openj9, hoping for slight performance improvement. After I changed java version to mojang's DEFAULT JAVA8, the mod worked just fine! I'm no expert in java so don't know exactly what was going on (no backward compat b/w java versions? or openj9 was bad?), but the problem solved.
Sorry for being just a personal troubleshooting rather than a bugfix of the mod, but your comment certainly help me solve the problem. Thank you!
